inventories/seapath_cluster_definition_example.yml

```diff
@@ -147,21 +147,6 @@ all:
                         cpumachinesnort: "6-11,18-23" # CPUs reserves for VMs non realtime
                         cpuovs: "23" # CPUs reserves for OVS
 
-        # VM group: use to deploy and manage guests on the cluster
-        VMs:
-            hosts:
-                test:
-                  live_migration: true
-                  preferred_host: "node3"
-                  migrate_to_timeout: 180
-                  enable: false
```

Comment:
Did you add this removed part in `seapath_vm_definition_example.yaml`? I don't see it anywhere in the file or in your PR

Reply:
No, because `seapath_vm_definition_example.yaml` is supposed to work with the template `guest.xml`.
The VM that I removed here is supposed to work with the seapath_rt and seapath_nort VM xml.
We should harmonize these xml to be more clear, but I will not do it in this PR.